dex/testing/btc/base-harness.sh

```diff
@@ -366,6 +367,67 @@ if [ ! "$CHAIN_LOADED" ]; then
   echo "**********************************************************************"
 fi
 
-# Reenable history and attach to the control session.
+# Reenable history
 tmux send-keys -t $SESSION:2 "set -o history" C-m
+
+if [ ! -z "$GODAEMON" ]; then
+  $GODAEMON --version &> /dev/null
+  DAEMON_INSTALLED=$?
+
+  if [ $DAEMON_INSTALLED -eq 0 ]; then
```

Comment:
A possibility that would eliminate the need to even install it would be using `go run`:

```
$  go run github.com/gcash/bchd@v0.19.0 -V
bchd version 0.19.0
```

So it could probably be set with:

```bash
export GODAEMON="go run github.com/gcash/bchd@v0.19.0"
```

and base-harness.sh could stay the same in case other assets can't use `go run`, like ltcd which has `replace` statements left in their go.mod.